corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/util.js

```diff
@@ -212,7 +211,8 @@ hqDefine("cloudcare/js/formplayer/utils/util", function () {
         this.forceLoginAs = options.forceLoginAs;
 
         this.setSelections = function (selections) {
-            this.selections = selections;
+            this.selections = selections || [];
+            sessionStorage.removeItem('selectedValues');
```

Comment:
@stephherbers I think this change might fix the same bug as https://github.com/dimagi/commcare-hq/pull/31949